modules/msgpack-c/6.1.0/patches/add_build_file.patch

```diff
@@ -0,0 +1,19 @@
+--- /dev/null
++++ BUILD.bazel
+@@ -0,0 +1,16 @@
++load("@rules_cc//cc:defs.bzl", "cc_library")
++
++cc_library(
++    name = "msgpack",
++    srcs = glob([
++        "src/*.c",
++        "include/**/*.h",
++        "include/**/*.hpp",
++    ]),
++    defines = ["MSGPACK_NO_BOOST"],
++    includes = [
```

Comment:
This target has `includes` set, but no `hdrs`. Is that correct?

Reply:
I have no idea, I took the BUILD file from https://github.com/envoyproxy/envoy/blob/main/bazel/external/msgpack.BUILD 

Maybe it's no right to do it this way. WDYT ?

Reply:
One way to know for sure that has other benefits is to add a small test module, which verified that the module can be used in a realistic scenario: https://github.com/bazelbuild/bazel-central-registry/blob/main/docs/README.md#test-module

Reply:
There are examples in the project : https://github.com/msgpack/msgpack-c/tree/cpp-6.1.0/example but none of them is using Bazel to be build. How can it be provided within the registry ?

Reply:
Hi @phlax,
As it is from envoy can you help with this question ?

Reply:
looking at how it has been setup in envoy here:

https://github.com/envoyproxy/envoy/blob/dc21640dff119e1365aa8286b7c34a1f0585e147/bazel/external/msgpack.BUILD#L3-L16

i think the `.h` and `.hpp` files _should_ be in `hdrs` - not sure if the `includes` is necessary - cc @ohadvano

Reply:
possibly its necessary to interact with `strip_include_prefix` but im guessing not